app/src/main/java/com/example/atheneum/utils/OwnerBooksAdapter.java

```diff
@@ -114,14 +100,29 @@ public void onClick(View v){
      * @param position
      */
     @Override
-    public void onBindViewHolder(MeasurementViewHolder holder, final int position)  {
+    public void onBindViewHolder(final MeasurementViewHolder holder, final int position)  {
         holder.titleTextView.setText(
                 ownerBooks.get(position).getTitle());
         holder.authorTextView.setText(
                 ownerBooks.get(position).getAuthor());
         holder.statusTextView.setText(
                 ownerBooks.get(position).getStatus().toString());
 
+        holder.bookItem.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v){
+//                Toast.makeText(parent.getContext(), "Test Click" + String.valueOf(vh.getAdapterPosition()), Toast.LENGTH_SHORT).show();
```

Comment:
Can remove some of these comments if you feel like it since that was dead code anyways.